### DISCUSSION TITLE
Is Motrix portable?

### DISCUSSION THREAD

**Discussion by o0Julia0o** [agalwood/Motrix#1196] [General]
Is Motrix portable, or have you to install it?

**Comment by easonlhello** [comment]
> Is Motrix portable, or have you to install it?
> 
> 

You need to install it

**Reply by vertigo220** [reply]
AFAICT this is incorrect. The main page recommends installing it for use of all features, which suggests that you don't _have_ to do so. Additionally, there are three separate downloads for Windows, one with "Setup" in the name and two without, and both of these do in fact run without installation (though I haven't tried them out). What I'm trying to figure out is what the difference is between these two (Motrix-version.exe and Motrix-version-win.zip). They seem more-or-less the same, but the exe's are different and the zip contains a bunch of other files and folders.
